glide.lock

```diff
@@ -25,18 +25,7 @@ testImports:
   subpackages:
   - assert
   - require
-- name: go.uber.org/atomic
-  version: 4e336646b2ef9fc6e47be8e21594178f98e5ebcf
 - name: go.uber.org/tools
   version: ce2550dad7144b81ae2f67dc5e55597643f6902b
   subpackages:
   - update-license
-- name: go.uber.org/zap
```

Comment:
This was a E2E test, so in theory this could be a test import. But it is ok to remove it, because we have other tests for functions unmarshalling.